README.md

```diff
@@ -285,4 +192,4 @@ This project is [MIT](./LICENSE) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
 
```

Comment:
Nice README 📖 but unfortunately your `MIT` link is broken 😞 , the reason being that the link points to a file called `LICENSE` when your file is called `LICENSE.md` 👀. Also, this second part is optional but consider removing that `NOTE` once you're done reading it, it's meant to be seen by you and only you. 💯